LICENSE.md

```diff
@@ -1,21 +1,124 @@
+platform; (iii) your use of the Ancillary Source Code will be uninterrupted, timely, secure, or error-free; (iv) the results that may
+be obtained from the use of the Ancillary Source Code will be accurate or reliable; (v) the quality of any products, services,
+information, or other material purchased or obtained by you through the Ancillary Source Code will meet your expectations; or
+(vi) any errors in the Ancillary Source Code will be corrected.  
+YOU EXPRESSLY UNDERSTAND AND AGREE THAT EPHESOFT SHALL NOT BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, CONSEQUENTIAL OR EXEMPLARY DAMAGES, INCLUDING BUT NOT
+LIMITED TO, DAMAGES FOR LOSS OF PROFITS, GOODWILL, USE, DATA OR OTHER INTANGIBLE LOSSES
+(EVEN IF EPHESOFT HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES) RELATED TO THE ANCILLARY
+SOURCE CODE, including, for example: (i) the use or the inability to use the Ancillary Source Code; (ii) the cost of
+procurement of substitute goods and services resulting from any goods, data, information or services purchased or
+obtained or messages received or transactions entered into through or from the Ancillary Source Code; (iii) unauthorized
+access to or alteration of your transmissions or data; (iv) statements or conduct of any third-party on the Ancillary Source
+Code; (v) or any other matter relating to the Ancillary Source Code.  
+Ephesoft reserves the right at any time and from time to time to modify or discontinue your use of the Ancilary
+Source Code, temporarily or permanently (or any part thereof) with or without notice. Ephesoft shall not be liable to you or to any third-party
+for any modification, price change, suspension or discontinuance of the Ancillary Source Code.
+
+## 8. Indemnity
+
+Licensee agrees to fully defend and indemnify Ephesoft and to hold Ephesoft harmless from and against any claims, suits or other proceedings, 
+actions, losses, costs, liabilities or expenses incurred in connection with the defense thereof (including reasonable attorney’s fees and paralegal fees), 
+in each case which may be imposed on, or incurred by, or asserted against Ephesoft in any way arising out of, directly or indirectly, 
+or relating to this Agreement, provided that Licensee shall not be liable for that portion of any such indemnification amount 
+resulting from Ephesoft’s gross negligence or intentional conduct.
+
+## 9. Term and Termination
+
+<ol type="a">
+    <li>This Agreement shall continue until terminated in accordance with this Section 9.</li>
+    <li>Upon a breach of any provision of this Agreement by Licensee, Ephesoft may immediately terminate this Agreement.</li>
+    <li>Ephesoft may terminate this Agreement for convenience and in respect of itself only at any time by giving sixty (60) days prior written notice to Licensee.</li>
+    <li>The provisions of this Agreement that are intended to survive shall continue in full force after termination of this Agreement.</li>
+</ol>
+
```

Comment:
This section has some HTML that needs to be removed.

Reply:
Instead of HTML you'll want to use MarkDown.  It is very easy and there are examples of ordered lists here: https://guides.github.com/features/mastering-markdown/

Basically, you'll do something like this:
```
1. Item 1
 1. Item2
 1. Item 3
    1. Item 3a
    1. Item 3b
```